debian/marquee/nginx/sites/url.spec.whatwg.org.conf

```diff
@@ -0,0 +1,28 @@
+server {
+    server_name url.spec.whatwg.org;
+    root /var/www/url.spec.whatwg.org;
+
+    ssl_certificate /etc/letsencrypt/live/url.spec.whatwg.org/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/url.spec.whatwg.org/privkey.pem;
+
+    include /etc/nginx/whatwg.conf;
+
+    location /interop/ {
+        return 410;
+    }
```

Comment:
Does this include any files within that directory?

Reply:
Yes, anything URL that begins with `/interop/`, which notably excludes `/interop` without a trailing slash. Since it's a directory people would have had to remove the slash manually from any URL they copied from a browser, so that seems OK.

I can make it whatever you want it to be, of course.